examples/sqlrooms-ai/tsconfig.app.json

```diff
@@ -6,6 +6,7 @@
     "lib": ["ES2020", "DOM", "DOM.Iterable"],
     "module": "ESNext",
     "skipLibCheck": true,
+    "sourceMap": true,
```

Comment:
Thanks!

Reply:
I also [added it to the base tsconfig](https://github.com/sqlrooms/sqlrooms/commit/549110f9362cd587886abd063c977aee6475f536)